packages/botfuel-dialog/src/index.js

```diff
@@ -69,7 +72,7 @@ module.exports = {
   FileCorpus,
   MemoryBrain,
   MongoBrain,
-  Nlu,
+  Nlu: BotfuelNlu,
```

Comment:
what is it for?

Reply:
Otherwise, it would be a breaking change.

Reply:
Will be the abstract class.

Reply:
done